_layouts/base.html

```diff
@@ -66,7 +66,11 @@
       apiKey: 'fdd89b9b35b9bd347f34111dbc1c6219',
       indexName: 'dita-ot',
       inputSelector: '#search input[type=text]',
-      debug: false // Set debug to true if you want to inspect the dropdown
+      algoliaOptions: {
+             hitsPerPage: 10,
```

Comment:
The trailing comma isn't strictly necessary here, but doesn't hurt and would be required if we add further options later, so it's worth keeping.